x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/enrich.tsx

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiLink } from '@elastic/eui';
+import {
+  FIELD_TYPES,
+  fieldValidators,
+  UseField,
+  Field,
+  ToggleField,
+  NumericField,
+  SelectField,
+  ValidationConfig,
+  useKibana,
+} from '../../../../../../shared_imports';
+
+import { FieldNameField } from './common_fields/field_name_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import { TargetField } from './common_fields/target_field';
+
+import { FieldsConfig, from, to } from './shared';
+
+const { emptyField, numberSmallerThanField, numberGreaterThanField } = fieldValidators;
+
+const maxMatchesValidators = {
+  max: numberSmallerThanField({
+    than: 128,
+    allowEquality: true,
+    message: i18n.translate(
+      'xpack.ingestPipelines.pipelineEditor.enrichForm.maxMatchesMaxNumberError',
+      { defaultMessage: 'This number must be less than 128.' }
+    ),
+  }),
+  min: numberGreaterThanField({
+    than: 0,
+    allowEquality: false,
+    message: i18n.translate(
+      'xpack.ingestPipelines.pipelineEditor.enrichForm.maxMatchesMinNumberError',
+      { defaultMessage: 'This number must be greater than 0.' }
+    ),
+  }),
+};
+
+const targetFieldValidator: ValidationConfig = {
+  validator: emptyField(
+    i18n.translate('xpack.ingestPipelines.pipelineEditor.enrichForm.targetFieldRequiredError', {
+      defaultMessage: 'A target field value is required.',
+    })
+  ),
+};
+
+const fieldsConfig: FieldsConfig = {
+  /* Required fields config */
+  policy_name: {
+    type: FIELD_TYPES.TEXT,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.enrichForm.policyNameFieldLabel', {
+      defaultMessage: 'Policy name',
+    }),
+    validations: [
+      {
+        validator: emptyField(
+          i18n.translate(
+            'xpack.ingestPipelines.pipelineEditor.enrichForm.policyNameRequiredError',
+            {
+              defaultMessage: 'A value is required.',
+            }
+          )
+        ),
+      },
+    ],
+  },
+
+  /* Optional fields config */
+  override: {
+    type: FIELD_TYPES.TOGGLE,
+    defaultValue: true,
+    deserializer: to.booleanOrUndef,
+    serializer: from.defaultBoolToUndef,
+    label: i18n.translate('xpack.ingestPipelines.pipelineEditor.enrichForm.overrideFieldLabel', {
+      defaultMessage: 'Override',
+    }),
+    helpText: i18n.translate(
+      'xpack.ingestPipelines.pipelineEditor.enrichForm.overrideFieldHelpText',
+      {
+        defaultMessage: 'If true, the processor can overwrite pre-existing field values.',
```

Comment:
This was my original suggestion, but `true` doesn't feel right here.
```suggestion
        defaultMessage: 'If enabled, the processor can overwrite pre-existing field values.',
```